e2e/specs/pyplot_kwargs.spec.ts

```diff
@@ -20,15 +20,17 @@
 describe("st.pyplot with kwargs", () => {
   before(() => {
     cy.visit("http://localhost:3000/");
+
+    // Wait for the site to be fully loaded
+    cy.contains("Done!", { timeout: 100000 }).should($els => {
```

Comment:
I'm wondering what `$els` is here since you didn't `.get()` any selectors

Reply:
`cy.contains` gets the DOM element containing the text sent by parameter, just like `get()`